.github/workflows/js.yml

```
@@ -0,0 +1,35 @@
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - main
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths:
+      - 'frontend/**'
+    paths-ignore:
```

actionlint (1.7.4)

both "paths" and "paths-ignore" filters cannot be used for the same event "pull_request". note: use '!' to negate patterns

(events)